apps/rush-lib/src/cli/actions/BaseRushAction.ts

```diff
@@ -83,6 +86,16 @@ export abstract class BaseRushAction extends CommandLineAction {
    */
   protected abstract run(): Promise<void>;
 
+  protected onDefineParameters(): void {
```

Comment:
I added this to `BaseRushAction` because a lot of actions needed to touch the Shrinkwrap file. I could try scoping it to *exactly* the set of actions that need it, I suppose.